generated/global-autoconfig-properties/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>generated-parent</artifactId>
```

Comment:
We'll also need to add `global-autoconfig-properties` as a child module into [spring-cloud-gcp/generated/pom](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/489874079dafc07f4d6810c097f08b2e1db83693/generated/pom.xml#L15-L16).